README.md

```diff
@@ -102,51 +102,43 @@ Alternatively, under root:
 
 You can add "-d" to build_full_frontend.sh to disable obfuscation and reduce build time.
 
-### Install and run pushpin for UI updates
+### Run Tracim for development
 
-Tracim relies on [pushpin](https://pushpin.org/docs/about/) for UI updates, so you need to setup pushpin.
+    ./run_dev_backend.sh
```

Comment:
May be useful to provide some docker compose helper stuff related to this to run postgres/mysql/mariadb for specific development but not test case.